libs/text-splitters/langchain_text_splitters/markdown.py

```diff
@@ -221,3 +222,119 @@ class HeaderType(TypedDict):
     level: int
     name: str
     data: str
+
+
+class ExperimentalMarkdownSyntaxTextSplitter:
```

Comment:
could we add some docs on the behavior of this textsplitter, especially as opposed to existing mardkown one

Reply:
Sure thing! Added some documentation, let me know what you think 😄

Reply:
Fixed linting errors